nemo_skills/pipeline/script.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Should we call this `run_script` instead? Don't have a strong opinion on this, but `ns run_script` seems more descriptive than just `ns script`

Reply:
Or run / run_cmd since it supports both script/module?

Reply:
I like `run_cmd` for both of the reasons you suggest. Updated in 2d7669dcff5ed5c7c42013b921f97371c2ac1f32